dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+  <ul>
+      <li>
+       If the asset is marked as "required", it checks the bid response to ensure the asset is part of the response
+      </li>
+      <li>
+       However, Prebid.js does not do any additional checking of a required asset beyond ensuring that it's included in the response; for example, it doesn't validate that the asset has a certain length or file size, just that that key exists in the response JSON
+      </li>
+    </ul>
+  </p>
+</div>
+
+<a name="native-image" />
+
+##### `adUnit.mediaTypes.native.image`
+
+{: .table .table-bordered .table-striped }
+| Name            | Scope    | Type                                  | Description                                                                                                                         |
+|-----------------+----------+---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------|
+| `required`      | Required | Boolean                               | Whether this asset is required.                                                                                                     |
+| `sizes`         | Optional | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                             |
+| `aspect_ratios` | Optional | Array[Object]                         | Instead of `sizes`, you can define allowed aspect ratios.  For properties, see [`image.aspect_ratios`](#native-image-aspectratios). |
+
+<a name="native-image-aspectratios" />
+
+###### `adUnit.mediaTypes.native.image.aspect_ratios`
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Type    | Description                                                                                          |
+|----------------+----------+---------+------------------------------------------------------------------------------------------------------|
+| `min_width`    | Optional | Integer | The minimum width required for an image to serve (in pixels).                                        |
+| `ratio_height` | Required | Integer | This, combined with `ratio_width`, determines the required aspect ratio for an image that can serve. |
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="native-title" />
+
+##### `adUnit.mediaTypes.native.title`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                          |
+|------------+----------+---------+------------------------------------------------------|
+| `required` | Required | Boolean | Whether a title asset is required on this native ad. |
+| `len`      | Optional | Integer | Length of title text, in characters.                 |
```

Comment:
"len" is meant to specify a maximum length, not an absolute length requirement.

Reply:
Thanks for the catch, updated in cecd7a4